wd-java-testng/pom.xml

```diff
@@ -30,7 +30,13 @@
     <dependency>
       <groupId>org.seleniumhq.selenium</groupId>
       <artifactId>selenium-java</artifactId>
-      <version>4.10.0</version>
+      <version>4.14.1</version>
+      <scope>test</scope>
+    </dependency>
+    <dependency>
+      <groupId>io.appium</groupId>
```

Comment:
Do the Android/iOS driver actually add any value here? Can't we just stick to the RemoteWebDriver?

Reply:
Removed